reduct/boolean-reduct/tests/helper-functions-test.metta

```diff
@@ -46,3 +46,42 @@
  ! (assertEqualToResult (nodeHasChildExp A) (False))
  ! (assertEqualToResult (nodeHasChildExp (AND)) (False))
  ! (assertEqualToResult (nodeHasChildExp AND) (False))
+
+
+; Test cases for getGsetAndChildren function.
+
+; Test case 1: Simple AND expression

```

Comment:
Can you please use the above format to write the test cases? You should specify the given input and expected output for the given function in an `assertEqual`